BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.m

```diff
@@ -60,10 +60,36 @@ - (BOXAPIOperation *)createOperation
 
 - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBlock
 {
+    [self performRequestWithCached:nil refreshed:completionBlock];
+}
+
+- (void)performRequestWithCached:(BOXItemArrayCompletionBlock)cacheBlock refreshed:(BOXItemArrayCompletionBlock)refreshBlock
+{
     BOOL isMainThread = [NSThread isMainThread];
     BOXAPIJSONOperation *folderOperation = (BOXAPIJSONOperation *)self.operation;
-
-    if (completionBlock) {
+    
+    if (cacheBlock) {
+        if (self.requestCache) {
```

Comment:
Yes, so if they want to use exclusively refreshed or cached data, they have the option to. ```performRequestWithCompletion:``` actually just calls this method with a nil cacheBlock.